BEIMA.DB.Schemas/building-schema.txt

```diff
@@ -1,5 +1,5 @@
 {
-    "buildingId": String,
+    "_id": String,
```

Comment:
Is this necessary? I'm concerned that this may cause conflicts with the actual building object, and it isn't immediately obvious to a new developer what this id is actually referencing. Is this a standard mongo thing?

Reply:
We would need to alter Device.cs to reflect this change as well.

Reply:
From the original PR comment:
> There was also a change to the DB schemas. I removed the "deviceTypeId" and "buildingId" fields from the DeviceType and Building schemas respectively, since that would actually just be the "_id" field that is automatically generated when inserted into the database. I added the "_id" field to all of the schemas since that is what we will expect to see stored in the DB.

buildingId and deviceTypeId are referring to the _id fields in the actual DeviceType and Building documents in Mongo. It does not make sense to have a buildingId field in the Building object when the _id field already exists and serves the same purpose. This is the same for deviceTypeId.

The only time we use the keys "buildingId" and "deviceTypeId" is when we are referencing the Building and DeviceType documents from another collection. In essence, a Device will keep using "buildingId" and "deviceTypeId" to reference the _id fields of the respective Building and DeviceType document.

Yes, this is standard Mongo behavior, "_id" is automatically inserted for any object and it can be used as a unique surrogate key for any document we store in Mongo.

The Device.cs class does not need to be changed. The class was already built to use the _id field.

Reply:
I totally misinterpreted all of this and forgot to read your PR comment. I thought we were removing the buildingId attribute from the device schema, not the building schema. This makes much more sense now. Ignore my comment, sorry for the confusion lol.

Reply:
No worries, it happens to all of us lol.